src/CRM/CivixBundle/Resources/views/Code/test-api.php.php

```diff
@@ -12,6 +12,7 @@
  * @group headless
  */
 class <?php echo $testClassName ?> extends \PHPUnit_Framework_TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
+  use \Civi\Test\Api3TestTrait;
```

Comment:
@eileenmcnaughton should we include the DocTrait as well?

Reply:
@seamuslee001 possibly - but this is like from July & I don't even remember it now so I don't want to scope creep it at the moment